portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -39,12 +39,19 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8156 | Test Summary: Current AC plan level is shown in a card"
-	@ignore = "true"
 	@priority = "5"
 	test CurrentACPlanLevelShownInCard {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Liferay DXP");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
+
+		ACNavigation.goToSettings();
 
-		// TODO LRAC-8156 CurrentACPlanLevelShownInCard pending implementation
+		ACSettings.goToUsage();
 
+		ACSettings.assertPlanDetails(planName = "Enterprise");
```

Comment:
Hey @AfonsoHSantos , I would suggest you to pass the values you want to validate regarding the plan, for example:
- planName = "Enterprise";
- individualsLimit = "100,000";
- pageViews = "60,000,000";
- usersLimit = "Unlimited";
- individualsAddOns = "5,000";
- pageViewsAddOns = "5,000,000";
- individualsAddOnsPrice = "500 USD";
- pageViewsAddOnsPrice = "250 USD";

So, you will validate these values on this assertPlanDetails macro.

Reply:
fixed